docs/contributing/install_dev.rst

```diff
@@ -162,25 +162,29 @@ running
 Now that you have plasmapy on your local computer and you have a
 virtual environment, you will want to "install" this development
 version of PlasmaPy along with its dependencies. Start by activating
-your virtual environment. Next you want install the PlasmaPy
-dependencies. One way to do this is to do
+your virtual environment.
 
-.. code-block:: bash
-
-   (plasmapy) user@name:~/programming/plasmapy$ pip install -r requirements/environment.txt
-
-Next, setup the development version of PlasmaPy which you just cloned
+Then, setup the development version of PlasmaPy which you just cloned
 by moving into the root directory of the cloned repo and running the
 following there:
 
-.. code-block:: bash
 
-   (plasmapy) user@name:~/programming/plasmapy/PlasmaPy$ pip install -e .
+.. code-block:: bash
 
+   (plasmapy) user@name:~/programming/plasmapy$ pip install --editable ".[tests,docs]"
 
 You should now be all set to run development versions of PlasmaPy
 modules via ``import PlasmaPy`` in your test scripts!
 
+The ``--editable`` flag will create a soft link to your cloned
+repository.  Any changes in Python code you make there will be there
+when you ``import plasmapy`` from an interactive session.
+
+   If you are not working within a virtual environment, this may end in
+   a permission error - this can be avoided via also adding the
+   ``--user`` flag. But seriously, use a virtual environment and spare
+   yourself the trouble.
```

Comment:
```suggestion
   If you are not working within a virtual environment, this may end in
   a permission error - this can be avoided via also adding the
   ``--user`` flag. But seriously, use a virtual environment and spare
   yourself the trouble.
```
```suggestion
.. tip::

   If you are not working within a virtual environment, this may end in
   a permission error — this can be avoided via also adding the
   ``--user`` flag. But seriously, use a virtual environment and spare
   yourself the trouble.
```